.circleci/config.yml

```diff
@@ -0,0 +1,78 @@
+version: 2
+jobs:
+  build:
+
+    # Primary command image where all commands run
+    docker:
+      - image: circleci/ruby:2.4.2-node-browsers
+        environment:
+          RAILS_ENV: test
+          ORANGELIGHT_HOST: localhost
+          ORANGELIGHT_USER: postgres
+
+      # Service container image available at 'host: localhost'
+      - image: circleci/postgres:9.6-alpine    
+        environment:
+          POSTGRES_USER: orangelight
+          POSTGRES_DB: orangelight_test
+          POSTGRES_PASSWORD: ""
+
+    working_directory: ~/orangelight
+
+    steps:
+      - checkout
+      #set timezone to est
+      - run:
+          name: Set Timezone to EST
+          command: echo 'America/New_York' = /etc/timezone
+      
+      # Restore bundle cache
+      - restore_cache:
+          keys:
+          - orangelight-{{ checksum "Gemfile.lock" }}
+          - orangelight-
+      
+      # Bundle install dependencies
+      - run:
+          name: Install dependencies
+          command: bundle check --path=vendor/bundle || bundle install --path=vendor/bundle --jobs 4 --retry 3
+      
+      - run: sudo apt install postgresql-client
+
+      - save_cache:
+          key: orangelight-{{ checksum "Gemfile.lock" }}
+          paths:
+            - ./vendor/bundle
+      - run:
+          name: Start Solr test instance
+          command: bundle exec rake solr:clean
+          command: bundle exec rake server:test
+          background: true
+
+      - run:
+          name: npm install
+          command: npm install
+
+      - run:
+          name: Yarn Install
+          command: yarn install --cache-folder ~/.cache/yarn
```

Comment:
Does the cached folder (`~/.cache/yarn`) persist? https://github.com/pulibrary/figgy/pull/959/files#diff-1d37e48f9ceff6d8030570cd36286a61R42 leads me to believe that, in order for this to persist between builds, additional config. options must be used...

Reply:
Thank you @jrgriffiniii . I will update the config.yml.